api/src/pcapi/core/bookings/api.py

```diff
@@ -802,12 +813,12 @@ def archive_old_bookings() -> None:
             Booking.query.join(Booking.stock, Stock.offer)
             .filter(date_condition)
             .filter(
-                offers_models.Offer.subcategoryId.in_(constants.FREE_OFFER_SUBCATEGORY_IDS_TO_ARCHIVE),
-                offers_models.Stock.price == 0,
+                offers_models.Stock.is_automatically_used,
             )
             .with_entities(Booking.id)
         )
     )
+    print(query_old_booking_ids)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏A supprimer